docs/source/1.0/spec/core/http-traits.rst

```diff
@@ -932,6 +934,67 @@ many HTTP client and server implementations enforce limits in practice.
 Carefully consider the maximum allowed length of each member that is bound to
 an HTTP query string or path.
 
+.. _httpQueryParams-trait:
+
+``httpQueryParams`` trait
+=========================
+
+Summary
+    Binds a map of key-value pairs to be added as query string parameters.
```

Comment:
nit: recommend changing to `Binds a map of key-value pairs to query string parameters`